partners_api/partners_api_tests/promo_tests.cpp

```diff
@@ -16,7 +16,7 @@ using namespace platform::tests_support;
 
 namespace
 {
-std::string const kTestId = "TestId";
+std::string const kTestId = "13835058055282357840";
```

Comment:
с первого взгляда может быть не очевидна связь этого id  с -4611686018427193776 в ResponseProvider.py
может коммент/статик ассерт?

Reply:
добавил коммент 
// It should be compatible with test id on test server side at tools/python/ResponseProvider.py:150.
// On server side this value is negative because of the prod server does not support unsigned values.